tests/output/webpage/output_test.js

```diff
@@ -149,18 +149,6 @@ describe("Linting", function() {
     );
 
     // Custom additions to Slowparse, to ban JavaScript stuff
-    failingTest("Audio element banned",
-        "<audio></audio>", [
-            {row: 0, column: 0, lint: {type: "ELEMENT_NOT_ALLOWED"}}
-        ]
-    );
-
-    failingTest("Video element banned",
-        "<video></video>", [
-            {row: 0, column: 0, lint: {type: "ELEMENT_NOT_ALLOWED"}}
-        ]
-    );
-
```

Comment:
These tests were failing because "audio" and "video" are in the ```disableTags``` array in webpage-output.js.  I figured that this was intentional, that's why I removed these tests.